src/commands/excelCommands/ExcelSyncCommand.ts

```diff
@@ -1,6 +1,8 @@
+import _ from "lodash"
 import xlsx from "node-xlsx"
 import * as yargs from "yargs"
 import { waitForConnectorToBeHealthy } from "../../utils/connectorUtils.js"
+import { parseString } from "../../utils/parseConfigStrings.js"
```

Comment:
That's confusing. The file contains the two methods `parseString` and `parseConfigStrings`. You import `parseConfigStrings` with the name `parseString`.